src/data/services/LmsApiService.js

```diff
@@ -30,6 +30,17 @@ class LmsApiService {
     return apiClient.get(enterpriseListUrl);
   }
 
+  static requestCodes(enterpriseName, emailAddress, numberOfCodes) {
```

Comment:
For functions that have more than one argument like this, it's usually a little cleaner to pass in an object with keys for the three variables, e.g.:

```javascript
static requestCodes({ enterpriseName, emailAddress, numberOfCodes }) {
  ...
}
```

That way, there's no confusion around the order of arguments passed into the function and it becomes more explicit.